### PR TITLE
add test for issue 405: CRTF reading with labelcolor defined

### PR DESCRIPTION
**Pull request by keflavich**
Draft PR: So far this is only the test, not a solution.

### CODE REVIEW

**Comment by keflavich** [comment]
OK now I think this is a solution.  We simply had some missing keywords in the metadata definitions.

**Comment by keflavich** [comment]
Solves #405

File: regions/io/crtf/tests/test_crtf.py [hunk 1]
```diff
@@ -129,7 +129,9 @@ def test_issue_312_regression():
 
 
 @pytest.mark.parametrize('filename', ['data/CRTFgeneral.crtf',
-                                      'data/CRTFgeneraloutput.crtf'])
+                                      'data/CRTFgeneraloutput.crtf',
+                                      'data/CRTF_labelcolor.crtf'
+                                     ])
```
Comment (keflavich) [inline regions/io/crtf/tests/test_crtf.py:134]:
```suggestion
                                      'data/CRTF_labelcolor.crtf'])
```

**Review by larrybradley — APPROVED** [review]
Thanks, @keflavich

**Comment by larrybradley** [comment]
At some point I want to refactor the CRTF I/O code (like I did for DS9 and FITS), but I'll have to do it on my own time since HST and JWST don't need CRTF.

**Pull request merged**